nh_eobs_backup/models/base_config_settings.py

```diff
@@ -0,0 +1,65 @@
+from openerp.osv import osv, fields
+
+
+class NHClinicalObservationBackupSettings(osv.TransientModel):
+    """
+    Add settings for the list of wards to backup
+    """
+
+    _inherit = 'base.config.settings'
+    _name = 'base.config.settings'
+
+    _columns = {
+        'locations_to_print': fields.many2many(
+            'nh.clinical.location',
+            domain=[['usage', '=', 'ward']],
+            string='Locations to print backup observation reports for'
+        )
+    }
+
+    def set_locations(self, cr, uid, id, context=None):
```

Comment:
Does setting of locations definitely work? I expected this method name to have to be `set_locations_to_print` according to [this guide](http://odoo-development.readthedocs.io/en/latest/dev/py/res.config.settings.html#module-xxx).